docs/roadmap-and-releases/releases/1.2.2.4-id-repository-and-1.2.1.2-id-authentication.md

```
@@ -4,31 +4,43 @@ hidden: true
 
 # 1.2.2.4 ID Repository & 1.2.1.2 ID Authentication
 
-**Release Version: v1.2.2.4 ID Repository & v1.2.1.2 ID Authentication**&#x20;
+**Release Version: v1.2.2.4 ID Repository & v1.2.1.2 ID Authentication**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)